cookbooks/aws-parallelcluster-config/recipes/directory_service.rb

```diff
@@ -49,6 +50,22 @@
     end
   end
 
+  # Password
+  password_secret_arn_str = node['cluster']['directory_service']['password_secret_arn']
+  password_secret_arn = parse_arn(password_secret_arn_str)
+  password_secret_service = password_secret_arn[:service]
+  password_secret_resource = password_secret_arn[:resource]
+  ldap_password =
+    if password_secret_service == "secretsmanager" && password_secret_resource.start_with?("secret")
```

Comment:
it would be useful to have some examples of the format of the password_secret_arn in the two cases

Reply:
Doing in next revision